screens/LoginandRegisterComponents/Login/reset.js

```diff
@@ -0,0 +1,118 @@
+import React from 'react';
+import { StyleSheet, Text, View,
+  TouchableOpacity, TextInput, Alert } from 'react-native';
+import { NavigationActions } from 'react-navigation'
+import { Container, Header, Content, Title } from 'native-base'
+import { KeyboardAwareScrollView } from 'react-native-keyboard-aware-scroll-view'
+import firebase from 'firebase'
+import 'firebase/firestore';
+
+export default class Reset extends React.Component {
+
+  static navigationOptions = {
+    headerStyle: {
+      backgroundColor: '#336E7B',
+    },
+    headerTintColor: '#FFFFFF'
+  }
+
+  constructor() {
+    super()
+    this.state = {
+      email: '',
+    };
+    this.firestore = firebase.firestore()
+  }
+
+  componentDidMount() {
+  }
+
+
+  checkFields() {
+    var errorMessage = ""
+    if (!this.state.email) {
+      errorMessage = errorMessage.concat("Email field is empty. ");
+    } else if (this.state.email.indexOf('@') < 1) {
+      errorMessage = errorMessage.concat("This email is invalid. ");
+    }
+    if (errorMessage) {
+      Alert.alert(errorMessage)
+    } else {
+      this.sendEmail()
+    }
+  }
+
+  sendEmail() {
+    const backAction = NavigationActions.back({
+      key: null
+    })
+    var auth = firebase.auth();
+    var emailAddress = this.state.email;
+    auth.sendPasswordResetEmail(emailAddress).then(function() {
+      Alert.alert(
+          'Password reset email has been sent.',
+          '',
+          [
+            {text: 'OK', onPress: () => {}}
+          ]
+      )
+    }).catch(function(error) {
```

Comment:
for some reason the app stops running when the email is not found, maybe there's documentation for this